pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1450,7 +1453,13 @@ func calculateRequestedVCPUs(cpuTopology *CPUTopology) uint32 {
 
 func CalculateNetworkQueueNumberAndGetCPUTopology(vmi *v1.VirtualMachineInstance) (uint32, *CPUTopology) {
 	cpuTopology := getCPUTopology(vmi)
-	return calculateRequestedVCPUs(cpuTopology), cpuTopology
+	queueNumber := calculateRequestedVCPUs(cpuTopology)
+
+	if queueNumber > multiQueueMaxQueues {
+		log.Log.V(3).Infof(fmt.Sprintf("Capped the number of queues to be the current maximum of tap device queues: %d", multiQueueMaxQueues))
```

Comment:
`Infof` with `fmt.Sprintf` is redundant.

Reply:
Right you are; removed it.